package/MDAnalysis/auxiliary/base.py

```diff
@@ -908,7 +908,7 @@ def _restart(self):
 
     def _reopen(self):
         """ Close and then reopen *auxfile*. """
-        if self.auxfile != None:
```

Comment:
this seems to reverse the logic?

Reply:
Wooops, I wanted to revert the change (not really associated to the PR) but I must have pressed `Ctrl-Z` only once. I will change it to `is not` if it's ok to leave this change within this PR.

Reply:
@richardjgowers – did @RMeli address all your issues?